src/extension.ts

```diff
@@ -508,6 +517,34 @@ class Device extends vscode.TreeItem {
             [this.shellPort.toString()]);
         term.show();
     }
+    
+    async captureScreenshot() {
+        const error = (message) => {
+            vscode.window.showErrorMessage("Error capturing screenshot: " + message);
+        }
+        
+        try {
+            const screenshotDirectory = await getSharedTempDir('ev3dev-screenshots');
+            const screenshotFile = `${screenshotDirectory}/ev3dev-${sanitizedDateString()}.png`;
+
+            const conn = await this.exec('fbgrab -');
+            const writeStream = fs.createWriteStream(screenshotFile);
+            writeStream.on('open', () => {
+                conn.stdout.pipe(writeStream);
+            });
+
+            writeStream.on('close', () => {
+                vscode.window.setStatusBarMessage(`Screenshot successfully captured`);

```

Comment:
And when it doesn't? I want the status bar message there to help us debug if/when the image doesn't actually open. That's the great thing about the status bar: it doesn't need to be added/removed and it doesn't take up additional space!